### ISSUE TITLE
Erroneous errors after nptyping 2.0 update

### ISSUE THREAD

**Issue by friggog** [PyCQA/flake8#1581]
### how did you install flake8?

```console
pip install flake8
```

### unmodified output of `flake8 --bug-report`

```json
{
  "dependencies": [],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.8.12",
    "system": "Windows"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "flake8-docstrings",
      "version": "1.6.0, pydocstyle: 6.1.1"
    },
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.8.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.4.0"
    }
  ],
  "version": "4.0.1"
}
```

#### what I expected to happen
nptyping type annotations such as `def func(arr: NDArray[Shape["N, *"], Int]) -> NDArray[Shape["N, *"], Int]:` not to throw an error, currently I get 2 errors:
`F722 syntax error in forward annotation 'N, *'`
`F821 undefined name 'N'`

#### sample code

```python
def func(arr: NDArray[Shape["N, *"], Int]) -> NDArray[Shape["N, *"], Int]:
    pass
```

#### commands ran

```console
$ flake8 t.py
...
```

**Comment by asottile** [comment]
please read the bold text in the issue template:

> **NOTE: flake8 is a linting framework and does not implement any checks**
>
> _if you are reporting a problem with a particular check, please track down the plugin which implements that check_
